src/database/queries/proposals/selectWithPagination.sql

```diff
@@ -20,9 +27,11 @@ WHERE
 			OR :search = ''
 		) THEN
 			TRUE
-		ELSE
-			proposal_field_values.value_search
+		ELSE (
```

Comment:
I know this is part of a series of `AND` clauses, but could it be out top as its own clause within the `WHERE`? Or even an `AND` on the `LEFT JOIN`? Doubt it makes any difference query-planner-wise but may be more readable.

Reply:
Update: LEFT JOIN isn't what we want (we don't just want base_field to be null: we want to actively reject the *proposal field values* where base field is forbidden).  I am also wary that adding it outside of the case statement could lead to proposals being rejected when they have forbidden base fields in contexts outside of search, which would be incorrect.

I'm gonna just leave it as originally written, since truly we are just saying "if a search is being invoked, ignore records that matched against forbidden base fields"